src/utils/component-versioning.js

```diff
@@ -9,24 +13,39 @@ const versionedTag = (tagName, version) => `${tagName}-${toKebabCase(version)}`;
 
 const extractDependencies = componentClass => {
   const { versions, tagName } = componentClass;
-  // extract all dependencies by comparing with master-component tag name
-  const dependencies = Object.keys(versions).filter(name => name !== tagName);
+  // extract all dependencies...
+  const dependencies = Object.keys(versions)
+    // ... by comparing with master-component tag name
+    .filter(name => name !== tagName)
+    // ... and sorting longest-first (e.g. axa-button-link comes before axa-button)
+    .sort((a, b) => b.length - a.length);
```

Comment:
Is sorting safe enough?  Or is this just one more step to be more safe beside the "{"- replace below?

Reply:
This is actually necessary, as you need to start with the longest names, to really make sure you do not process long names with shorter ones.

Example:
`<axa-button` HAS TO BE processed after `<axa-button-link` with this approach (otherwise the wrong component will be versioned) and this sorting makes sure of that.

Reply:
Raphael is right here.